web/components/admin/FcAdminPermissions.vue

```diff
@@ -98,9 +98,9 @@ export default {
         value: 'UNIQUE_NAME',
         text: 'User',
       },
-      ...AuthScope.enumValues.map(({ name }) => ({
+      ...AuthScope.enumValues.map(({ name, description }) => ({
```

Comment:
This was scope creep... we were using the name (MVCR_READ) rather than the description (View and Download MVCRs) in the table headers... so I changed it.